components/alert/alert.css

```diff
@@ -1,25 +1,37 @@
 .alert {
   border: 0.5px solid;
```

Comment:
I think 1 px is the least possible unit please use integers not floating point number when defining px.

Reply:
sure, thanks for pointing it out.